app/models/service.rb

```diff
@@ -88,6 +89,7 @@ class Service < ApplicationRecord
   default_value_for :display, false
   default_value_for :retired, false
   default_value_for :initiator, 'user'
+  default_value_for :lifecycle_state, 'unprovisioned'
```

Comment:
@lfu I know we discussed using 'unprovisioned', but I'm wondering if there is another value we could use here?
@mkanoor Thoughts?